contracts/controller/src/prices.rs

```diff
@@ -24,42 +24,43 @@ impl Contract {
 
 #[cfg(test)]
 mod tests {
-
-    use near_sdk::AccountId;
-    use near_sdk::test_utils::test_env::{alice, bob, carol};
     use assert_matches::assert_matches;
+    use near_sdk::test_utils::test_env::{alice, bob};
 
-    use crate::{Config, Contract};
+    use super::*;
```

Comment:
@dmytro-shash do we really need import everything from outer scope?

In general stick to the rule allow \ import \ write \ etc. as less as reasonably possible